extensions/helix/package.build.ts

```diff
@@ -0,0 +1,217 @@
+// Save to package.json
+// ============================================================================
+
+import { Builder, generateIgnoredKeybinds } from "../../meta";
+import * as fs from "fs/promises";
+import { SelectionBehavior } from "../../src/api";
+
+const version = "0.1.0",
+      preRelease = 1,
+      preReleaseVersion = version.replace(/\d+$/, "$&" + preRelease.toString().padStart(3, "0"));
+
+export const pkg = (modules: Builder.ParsedModule[]) => ({
+
+  // Common package.json properties.
+  // ==========================================================================
+
+  name: "dance-helix-keybinds",
+  description: "Helix keybindings for dance",
+  version,
+  license: "ISC",
+
+  author: {
+    name: "Grégoire Geis",
+    email: "opensource@gregoirege.is",
+  },
```

Comment:
Dance doesn't really have an "author list" and I don't know much how it works, but it seems like [`package.json` can have one `author` and a list of `contributors`](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#people-fields-author-contributors).

If you don't mind, I think it would be fair to add yourself to a `contributors` array here.